internal/api/v1beta1/receiver.go

```diff
@@ -66,7 +69,7 @@ func (s *GRPCServer) CreateReceiver(ctx context.Context, req *sirenv1beta1.Creat
 }
 
 func (s *GRPCServer) GetReceiver(ctx context.Context, req *sirenv1beta1.GetReceiverRequest) (*sirenv1beta1.GetReceiverResponse, error) {
-	rcv, err := s.receiverService.Get(ctx, req.GetId(), receiver.GetWithData(true))
+	rcv, err := s.receiverService.Get(ctx, req.GetId(), receiver.GetWithData())
```

Comment:
Dex will make sure to send `parent_id` which is a `slack` type receiver.

But from Siren perspective, should it checks if the parent is compatible with given `type` (in this case `slack_channel`)?

Or that is something Siren requires the client to make sure?

Reply:
There is no hard check on that, but the `PreHookDBTransformConfig` function will always be called before inserting any data into DB `plugins/receivers/slackchannel/service.go` and inside the function we do validate required configs.